spec/models/comic_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [Sider](https://sider.review) -->